<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnPopulate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>105</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Populate Table</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/gear.png</normaloff>images/gear.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>10</y>
     <width>105</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Row(s)</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/cross.png</normaloff>images/cross.png</iconset>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>330</width>
     <height>192</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/door-open-out.png</normaloff>images/door-open-out.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
